main_app/templates/base.html

```diff
@@ -16,37 +16,17 @@
     </head>
     <body>
         <header class="">
-            <ul id="adddropdown" class="dropdown-content">
```

Comment:
dropdown display links removed. These lines are needed for the line from 32 to 40 to work

Reply:
Dropdown display links are back. Pending integration with responsive navbar code.